### ISSUE TITLE
Cannot run builds locally due to included .DS_Store

### ISSUE THREAD

**Issue by BastianZim** [conda-forge/staged-recipes#13531]
Hi,

I cannot run `CONFIG=win64 ./.scripts/run_docker_build.sh` on my Mac because I always end up with the error `ValueError: Specified folder /home/conda/staged-recipes-copy/recipes/.DS_Store does not exist` and I don't know where this comes from.

`.DS_Store` is included in my _.gitignore_ but it's still pulling it from somewhere. Does anyone know where this could come from?

Thanks!

Also, apologies if this is not the right repo, wasn't sure where to post this.

**Comment by BastianZim** [comment]
Pinging @conda-forge/staged-recipes

**Comment by chrisburr** [comment]
Finder creates `.DS_Store` files automatically, they can be safely deleted.

If you're interested in making a PR to fix it, I guess this function is the one you need to change: https://github.com/conda-forge/staged-recipes/blob/1f5fa77986c4a2bda208598b1f98b436b923c702/.travis_scripts/create_feedstocks.py#L39-L51

**Comment by BastianZim** [comment]
That's it, thank you! Apparently, [Apple removed](https://stackoverflow.com/a/54937530/9599354) showing `.DS_Store` when listing normal dot-files. Managed to remove it and now it works.

I can definitely open a PR for that but I just wanted to confirm if that's the only place? I also found https://github.com/conda-forge/staged-recipes/blob/1f5fa77986c4a2bda208598b1f98b436b923c702/.ci_support/build_all.py#L26-L32 which also lists some directories. 

If there is some sort of "overall" function I would start there, otherwise I can just create one for each.
